assets/css/filtersection.css

```diff
@@ -55,7 +38,353 @@
+        z-index: 5;
+        height: 40px
+    }
+
+    #mobile-nav .toggle-close {
+        position: fixed;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        background: 0 0;
+        z-index: 0;
+        display: none
+    }
+
+    #mobile-nav .toggle {
+        float: left;
+        width: 18px;
+        height: 20px;
+        margin-right: 14px;
+        margin-left: 16%;
+        background-image: url(../../img/toggle.png);
+        background-image: url(https://nbadraft.theringer.com/img/toggle.svg);
+        background-size: 100%;
+        background-position: center center;
+        -webkit-transition: margin-left 125ms linear;
+        -moz-transition: margin-left 125ms linear;
+        -ms-transition: margin-left 125ms linear;
+        -o-transition: margin-left 125ms linear;
+        transition: margin-left 125ms linear
+    }
+
+    #mobile-nav .current-sort {
+        font-weight: 700;
+        text-transform: uppercase;
+        font-size: 12.5px;
+        letter-spacing: .1em;
+        padding: 2px 0
+    }
+
+    #mobile-nav .nav-contents {
+        display: none
+    }
+
+    #mobile-nav.open {
+        background-position: right 15px top 17px;
+        bottom: 0;
+        margin-bottom: 0
+    }
+
+    #mobile-nav.open .filter-logo {
+        left: -16.5%
+    }
+
+    #mobile-nav.open .current-sort {
+        display: none
+    }
+
+    #mobile-nav.open .toggle {
+        margin-left: 15.5px;
+        /* margin-left: 100%;  */
+        /* float: right;  */
+    }
+
+    #mobile-nav.open .nav-contents {
+        position: relative;
+        display: block;
+        z-index: 1
+    }
+
+    #mobile-nav.open .nav-contents .current-sort {
+        display: block
+    }
+
+    #mobile-nav.open .nav-contents .nav-actions {
+        padding: 35px 0 10px;
+        width: 100%
+    }
+
+    #mobile-nav.open .nav-contents .nav-actions:after {
+        content: '';
+        clear: both;
+        display: table
+    }
+
+    #mobile-nav.open .nav-contents .nav-actions>div {
+        float: left;
+        width: 50%;
+        font-weight: 500;
+        font-size: 14px;
+        margin: 0 0 0 40px
+    }
+
+    #mobile-nav.open .nav-contents .nav-actions a {
+        display: block;
+        color: var(--gray);
+        margin-bottom: 3px;
+        font-weight: 100;
+        font-size: 15px
+    }
+
+    #mobile-nav.open .nav-contents .nav-actions input {
+        margin-left: 15px;
+    }
+
+    #mobile-nav.open .nav-contents .nav-actions a.active {
+        color: var(--lightred);
+    }
+
+    #mobile-nav.open .toggle-close {
+        display: block
+    }
+}
+
+#sticky-phantom {
+    background: 0 0;
+    color: #fff;
+    font-size: 46.5px;
+    text-align: center;
+    width: 100vw;
+    visibility: hidden
+}
+
+@media (max-width:1024px) {
+    #sticky-phantom {
+        font-size: 74.5px
+    }
+}
+
+@media (max-width:767px) {
+    #sticky-phantom {
+        font-size: 31px
+    }
```

Comment:
I encourage you to check out BEM notation for your CSS classes.

http://getbem.com/introduction/